packages/components/package.json

```diff
@@ -96,6 +105,8 @@
     "@itwin/itwinui-react": "catalog:itwinui",
     "@itwin/presentation-common": "catalog:itwinjs-core",
     "@itwin/presentation-frontend": "catalog:itwinjs-core",
+    "@itwin/unified-selection": "workspace:^",
```

Comment:
It should be enough to have only in dependencies `@itwin/unified-selection`

Reply:
removed